nautobot/extras/jobs.py

```diff
@@ -993,7 +1000,7 @@ def run_job(data, request, job_result_pk, commit=True, *args, **kwargs):
 
     try:
         data = job_class.deserialize_data(data)
-    except ObjectDoesNotExist as e:
+    except Exception as e:
```

Comment:
Can you please add an appropriate unit test case for cases that result in exceptions other than ObjectDoesNotExist?